core/src/main/java/com/digitalpebble/stormcrawler/protocol/AbstractHttpProtocol.java

```diff
@@ -96,4 +99,14 @@ protected static String getAgentString(String agentName,
         return buf.toString();
     }
 
+    /**
+     * Return new instance of {@link SimpleDateFormat} to format dates in HTTP
+     * headers, cf. <a href=
+     * "https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.3">sec3.3 in
+     * RFC 2616</a>.
+     */
+    public static SimpleDateFormat getHttpDateFormat() {
```

Comment:
Sure, but it is not used at all within the package. We don't check that the values sent in the protocol match this format, we just take the Strings as granted. I understand the logic but it is more practical and understandable to put it in the scheduler class I think.